src/librustdoc/html/render.rs

```diff
@@ -1340,10 +1315,17 @@ fn write_minify_replacer<W: Write>(
                         .into();
                     tokens.apply(|f| {
                         // We add a backline after the newly created variables.
-                        minifier::js::aggregate_strings_into_array_with_separation(
+                        minifier::js::aggregate_strings_into_array_with_separation_filter(
                             f,
                             "R",
                             Token::Char(ReservedChar::Backline),
+                            // This closure prevents crates' name to be aggregated. It allows to not
```

Comment:
"This closure prevents crates' names from being aggregated."

I'm not sure what the second sentence means. And this doesn't explain how the OpenBracket or searchIndex check work.

Reply:
> I'm not sure what the second sentence means. And this doesn't explain how the OpenBracket or searchIndex check work.

Ah? I'll reword it then...